### ISSUE TITLE
Duplicate Scene, context menu, create new scene in wrong folder

### ISSUE THREAD

**Issue by Bladum** [godotengine/godot#15576]
**Godot version:**
3.0 Beta 2

**Issue description:**
Duplication of Scene create new Scene in root folder of project instead of same folder as parent

**Steps to reproduce:**
Duplicate Scene somewhere deep in Folder structure using context menu. Add its new name. It will be then created on the top of folder structure. Need to move file manually.

**Comment by Paulloz** [comment]
I quickly tested this and it seems to work at 2c17af0.
```
Duplicating res://foo/bar/foobar.tscn -> res://foo/bar/foobarbis.tscn
```

**Comment by volzhs** [comment]
duplicated issue with #15206 and fixed by #15224
